ios/RCCViewController.m

```diff
@@ -644,6 +644,16 @@ -(void)setStyleOnAppearForViewController:(UIViewController*)viewController appea
       self.navigationItem.titleView.clipsToBounds = YES;
     }
   }
+  
+  if ([[[UIDevice currentDevice] systemVersion] floatValue] >= 11.0) {
+    NSNumber *prefersLargeTitles = self.navigatorStyle[@"prefersLargeTitles"];
+    BOOL prefersLargeTitlesBool = prefersLargeTitles ? [prefersLargeTitles boolValue] : NO;
+    if (prefersLargeTitlesBool) {
+      [self.navigationController.navigationBar setPrefersLargeTitles:YES];
```

Comment:
These setters can be written as `self.navigationController.navigationBar.prefersLargeTitles = prefersLargeTitlesBool ` as well, to use Objective-C 2.0 syntax, as well as reuse the `prefersLargeTitlesBool` variable to match the logic.